apps/sms/views/inbox/js/inbox.js

```diff
@@ -39,6 +39,11 @@ var InboxView = {
   // Set to |true| when in edit mode
   inEditMode: false,
 
+  /**
+   * Indicates whether user should be notified about newly saved draft.
+   */
+  notifyAboutSavedDraft: false,
```

Comment:
Yes, you're right, it doesn't work in split mode. This case is just a small part of the problem we need to solve :)

It sounds like we need to manage some sort of "state" between view transitions (especially the case when we use BF cache) in split mode. Maybe on "pageshow" Inbox can kind of request missed events from DraftsClient/ConversationClient, so SharedWorker can be a host for this state.

What do you have in mind?